modules/thunder_demo/content/user/09ec113d-32a1-4dcf-80d9-fc78069ce7b1.json

```diff
@@ -0,0 +1,101 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/default\/user\/4?_format=hal_json"
+        },
+        "type": {
+            "href": "http:\/\/drupal.org\/rest\/type\/user\/user"
+        }
+    },
+    "uid": [
+        {
+            "value": 4
+        }
+    ],
+    "uuid": [
+        {
+            "value": "09ec113d-32a1-4dcf-80d9-fc78069ce7b1"
+        }
+    ],
+    "langcode": [
+        {
+            "value": "en",
+            "lang": "en"
+        }
+    ],
+    "preferred_langcode": [
+        {
+            "value": "en"
+        }
+    ],
+    "preferred_admin_langcode": [
+        {
+            "value": "en"
+        }
+    ],
+    "name": [
+        {
+            "value": "kayne"
```

Comment:
I guess this should be Kanye :) - no it is just Ye now.

Reply:
Ticket is defined on 24.09.2018. It as legacy naming. 🤣